frontend/public/components/RBAC/edit-rule.jsx

```diff
@@ -310,15 +314,18 @@ const EditRule = connect(stateToProps, { getResources: k8sActions.getResources }
             <div className="row">
               <div className="col-xs-12">
                 <p className="text-secondary">
-                  Each role is made up of a set of rules, which defines the type of access and
-                  resources that are allowed to be manipulated.
+                  {t(
+                    'edit-rule~Each role is made up of a set of rules, which defines the type of access and resources that are allowed to be manipulated.',
+                  )}
                 </p>
               </div>
             </div>
 
             <div className="row rule-row">
               <div className="col-xs-2">
-                <strong>{this.kind.label} Name</strong>
+                <strong>
+                  {this.kind.label} {t('edit-rule~Name')}
```

Comment:
I'd make two messages with the kind hard-coded, one for ClusterRoles and one for Roles. This will be better for translation.